CDP4Reporting/SubmittableParameterValues/SubmittableParameterValue.cs

```diff
@@ -48,14 +58,63 @@ public class SubmittableParameterValue
         public string Path { get; private set; }
 
         /// <summary>
-        /// 
+        /// Indicates that the option part of the path should match exactly
+        /// </summary>
+        public bool IsExactOptionPath { get; private set; }
+
+        /// <summary>
+        /// Creates a new instance of <see cref="SubmittableParameterValue"/>
         /// </summary>
         /// <param name="path">
         /// The <see cref="NestedParameter.Path"/> of the <see cref="ParameterBase"/> in the Product Tree.
         /// </param>
-        public SubmittableParameterValue(string path)
+        /// <param name="isExactOptionPath">
+        /// Indication if the path is an exact Option path;
+        /// </param>
+        public SubmittableParameterValue(string path, bool isExactOptionPath)
         {
             this.Path = path;
+            this.IsExactOptionPath = isExactOptionPath;
+        }
+
+        /// <summary>
+        /// Tries to extract a value from a comma separated string that contains
+        /// key/value pairs divided by an '=' character
+        /// </summary>
+        /// <param name="extractFrom">The <see cref="string"/> to extract the value from</param>
+        /// <param name="key">The key</param>
+        /// <param name="value">The value</param>
+        /// <returns>true if the path was found, otherwise false</returns>
+        public static bool TryExtractValue(string extractFrom, string key, out string value)
```

Comment:
can this go to StringExtensions?

Reply:
This is not generic enough for StringExtensions. It is specifically used withing the context of the class it is located in (SubmittableParameterValue).